Extension/package.json

```diff
@@ -282,2546 +282,2593 @@
+            ],
+            "default": "insert",
+            "description": "%c_cpp.configuration.vcFormat.space.aroundTernaryOperator.description%",
+            "scope": "resource"
+          },
+          "C_Cpp.vcFormat.wrap.preserveBlocks": {
+            "type": "string",
+            "enum": [
+              "oneLiners",
+              "allOneLineScopes",
+              "never"
+            ],
+            "markdownEnumDescriptions": [
+              "%c_cpp.configuration.vcFormat.wrap.preserveBlocks.oneLiners.markdownDescription%",
+              "%c_cpp.configuration.vcFormat.wrap.preserveBlocks.allOneLineScopes.markdownDescription%",
+              "%c_cpp.configuration.vcFormat.wrap.preserveBlocks.never.markdownDescription%"
+            ],
+            "default": "oneLiners",
+            "description": "%c_cpp.configuration.vcFormat.wrap.preserveBlocks.description%",
+            "scope": "resource"
+          }
+        }
+      },
+      {
+        "title": "Doxygen",
+        "properties": {
+          "C_Cpp.doxygen.generateOnType": {
+            "type": "boolean",
+            "default": true,
+            "description": "%c_cpp.configuration.doxygen.generateOnType.description%",
+            "scope": "resource"
+          },
+          "C_Cpp.doxygen.generatedStyle": {
+            "type": "string",
+            "enum": [
+              "///",
+              "/**",
+              "/*!",
+              "//!"
+            ],
+            "default": "///",
+            "description": "%c_cpp.configuration.doxygen.generatedStyle.description%",
+            "scope": "resource"
+          },
+          "C_Cpp.doxygen.sectionTags": {
+            "type": "array",
+            "default": [
+              "attention",
+              "tparam",
+              "param",
+              "result",
+              "returns",
+              "retval",
+              "exceptions",
+              "deprecated",
+              "warning",
+              "note"
+            ],
+            "items": {
+              "type": "string",
+              "enum": [
+                "attention",
+                "author",
+                "authors",
+                "bug",
+                "copyright",
+                "date",
+                "deprecated",
+                "details",
+                "exceptions",
+                "invariant",
+                "note",
+                "param",
+                "pre",
+                "post",
+                "remark",
+                "remarks",
+                "result",
+                "returns",
+                "retval",
+                "since",
+                "tparam",
+                "test",
+                "todo",
+                "version",
+                "warning"
+              ]
+            },
+            "description": "%c_cpp.configuration.doxygen.sectionTags.description%",
+            "scope": "resource"
+          }
+        }
+      },
+      {
+        "title": "Default",
```

Comment:
This one should be "IntelliSense Configuration" (if that fits in the UI).  "Configuration" if it needs to be shorter.